src/main/webapp/css/home-page.css

```
@@ -0,0 +1,46 @@
+
+.carousel-control-prev {
+    margin-left: -125px;
+    filter: invert(100%);
+  }
+  .carousel-control-next {
+    margin-right: -125px;
+    filter: invert(100%);
+  }
+  
+  .carousel-indicators {
+    filter: invert(100%);
+    margin-bottom: -50px;
+  }
+
+.home-deal-img {
+    height: 270px;
+  }
+
+  #search-btn {
+    width: 20%;
+    padding: 10px;
+    background: #2196F3;
+    color: white;
+    font-size: 17px;
+    border: 1px solid grey;
+    border-left: none;
+    cursor: pointer;
+  }
+
+  
+  #search-btn:hover {
+    background: #0b7dda;
+  }
+
+  #search-input {
+    padding: 10px;
+    font-size: 17px;
+    border: 1px solid grey;
+    width: 50%;
+    background: #f1f1f1;
+  }
+
+  form {
+      font-size: 0px;
```

```suggestion
      display: flex;
      justify-content: center;
```
https://css-tricks.com/fighting-the-space-between-inline-block-elements/
the font-size 0 feels weird